load-from-dir.js

```diff
@@ -28,7 +28,7 @@ function loadFromDir (dir, top) {
     if (fileinfo.isSymbolicLink()) {
       var dest = fs.readlinkSync(filepath)
       if (/^(?:[a-z]\:)?[\\/]/i.test(dest)) {
-        dest = '/' + path.relative(top, dest)
+        dest = path.sep + path.relative(top, dest)
```

Comment:
It's plausible that we should be normalizing the `path.relative` results however.